ios/MaasCore/Sources/MaasTheme/Theme/Shadow.swift

```diff
@@ -0,0 +1,26 @@
+public extension View {
+
+    func shadowStyle(_ shadowStyle: ShadowStyle) -> some View {
+        ModifiedContent(content: self, modifier: ShadowStyleModifier(shadowStyle: shadowStyle))
+    }
+}
+
+private struct ShadowStyleModifier: ViewModifier {
+
+    let shadowStyle: ShadowStyle
+
+    func body(content: Content) -> some View {
+
+        var modifiedContent = AnyView(content)
+
+        Kotlin(self.shadowStyle).shadows.forEach {
```

Comment:
This `Kotlin` wrapper here is not necessary as you wrap each individual shadow below with `Kotlin($0)`, right?
```suggestion
        self.shadowStyle.shadows.forEach {
```